fixtures/workers-with-assets-and-service-bindings/tests/index.test.ts

```diff
@@ -55,7 +59,7 @@ describe.each(devCmds)(
 			describe("Service binding to default export", () => {
 				// this currently incorrectly returns the User Worker response
 				// instead of the Asset Worker response
-				it.fails(
+				failsIf(!args.length)(
```

Comment:
this is a good thing! this means that our Proxy Worker solved this use case

Reply:
should the comment above be updated as well?

adding something like `unless --x-assets-rpc is provided` at the end?